### PR TITLE
Updated twitter logo in share button and removed twitter logo from footer #8599

### LINKED ISSUE #8599: Updating Twitter logo to X
Twitter logo has been changed to X.  But the website's logo hasn't updated.

### Proposal & Constraints

The current logo looks like: 
Screenshot 2023-12-08 114548

After changing the logo will look like:
Screenshot 2023-12-08 114519

If the issue needs to be solved. 
**please assign me**, would like to work on it.

### PR DESCRIPTION
**Pull request by parmishh**
Closes #8599
1.Updated twitter logo in share button
2. removed twitter logo from footer
Video of updates:

https://github.com/internetarchive/openlibrary/assets/91942072/0178253f-1468-4867-a23a-92b622350e82

### CODE REVIEW

**Comment by parmishh** [comment]
@mekarpeles made a PR

**Comment by mekarpeles** [comment]
Thank you @parmishh -- team, we may want a smaller svg for the `X` icon to reduce load time

**Comment by mekarpeles** [comment]
I think for the time being, since twitter/x is still a platform Open Library is actively using for communications, reluctantly, the penalty of removing (to patrons) and accessing updates may be worse than keeping.

Apologies for the change of mind -- switching the logo is still the right choice, for now let's re-add the icon where it was, if it's not too much trouble.

**Comment by RayBB** [comment]
@parmishh are you still able to work on this?

File: openlibrary/templates/lib/nav_foot.html [hunk 1]
```diff
@@ -46,7 +46,6 @@ <h2>$:_('Help')</h2>
           <li><a href="https://github.com/internetarchive/openlibrary/releases" title="$_('Release Notes')">$_('Release Notes')</a></li>
         </ul>
         <aside>
```
Comment (mekarpeles) [inline openlibrary/templates/lib/nav_foot.html:48]:
```suggestion
        <aside>
          <a class="footer-icon" title="Twitter" href="https://x.com/OpenLibrary"><img src="/static/images/twitter.svg" alt="" loading="lazy"></a>
```

**Comment by mekarpeles** [comment]
I think the x icon needs a treatment of `filter: invert(100%);` to be visible, but otherwise things are looking okay.

**Comment by mekarpeles** [comment]
Before:
Screenshot 2024-06-15 at 10 28 05 AM

After:
Screenshot 2024-06-15 at 10 27 57 AM

File: openlibrary/templates/lib/nav_foot.html [hunk 2]
```diff
@@ -46,7 +46,7 @@ <h2>$_("Help")</h2>
           <li><a href="https://github.com/internetarchive/openlibrary/releases" title="$_('Release Notes')">$_("Release Notes")</a></li>
         </ul>
         <aside>
-          <a class="footer-icon" title="$_('Twitter')" href="https://twitter.com/OpenLibrary"><img src="/static/images/tweet.svg" alt="" loading="lazy"></a>
+          <a class="footer-icon invert" title="$_('Twitter')" href="https://x.com/OpenLibrary"><img src="/static/images/twitter.svg" alt="" loading="lazy"></a>
```
Comment (mekarpeles) [inline openlibrary/templates/lib/nav_foot.html:49]:
```suggestion
          <a class="footer-icon" title="$_('Twitter')" href="https://x.com/OpenLibrary"><img class="invert" src="/static/images/twitter.svg" alt="Open Library Twitter X logo" loading="lazy"></a>
```

**Comment by mekarpeles** [comment]
While I feel bad, I think for now it makes sense to close this issue and keep things the way they are for now and closing #8599

**Pull request closed without merging**